pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -41,6 +41,9 @@ let
         (whenBetween "5.2" "5.18" yes)
       ];
       DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT = whenAtLeast "5.18" yes;
+      # Reduced debug info conflict with BTF and have been enabled in
+      # aarch64 defconfig since 5.13
+      DEBUG_INFO_REDUCED        = no;
```

Comment:
For linux 4.19, I get: `error: unused option: DEBUG_INFO_REDUCED` can you add a lower bound to avoid breaking older kernels?

Reply:
Hmm, I didn't because the option appeared in v2.6.36 (! https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=d6f4ceb796ebf1a8c8a9ad4a8ea0d181aaec7de6 )

Since I'm not sure why it's not used for you and we only care about 5.13+ kernels I've just arbitrarily made that the limit, but it's a bit weird. Also, why didn't the config system complain about DEBUG_INFO_BTF not being asked when it was masked by DEBUG_INFO_REDUCED?.. I don't quite get this configuration system, sorry.